script/make_eol_xml

```diff
@@ -84,8 +84,8 @@ end
 ################################################################################
 
 if testing
-  LOCAL_URL = "https://localhost:3000/api/observations?id=1"
-  REMOTE_URL = "https://mushroomobserver.org/api/observations?id=1"
+  LOCAL_URL = "https://localhost:3000/api2/observations?id=1"
```

Comment:
Should this (and line 90) be `http:`?

Reply:
1. We can't go to `https://localhost...`. So things would blow up if someone tried that. 
2. I don't know if we are really using the eol code currently. @mo-nathan is the best person to answer that.

Reply:
Agreed, the local server on the dev VM isn't secure.

Reply:
Changed to `http://localhost...` in both places